packages/create-sitecore-jss/src/initializers/nextjs-xmcloud/index.ts

```diff
@@ -9,19 +9,25 @@ import {
   incompatibleAddonsMsg,
 } from '../../common';
 import { removeDevDependencies } from './remove-dev-dependencies';
+import { sharedPrerender, Prerender } from '../nextjs/prompts';
+
+type ExtendedClientAppArgs = ClientAppArgs & {
```

Comment:
There's already a type for args with `prerender`, NextjsArgs in nextjs/prompts.ts
Does the issue still occur if you use that?

Reply:
Yes we can.
Updated.